Algorithm/QCAlgorithm.History.cs

```diff
@@ -225,13 +225,15 @@ private bool TryGetWarmupHistoryStartTime(out DateTime result)
         /// <param name="extendedMarketHours">True to include extended market hours data, false otherwise</param>
         /// <param name="dataMappingMode">The contract mapping mode to use for the security history request</param>
         /// <param name="dataNormalizationMode">The price scaling mode to use for the securities history</param>
+        /// <param name="contractDepthOffset">The continuous contract desired offset from the current front month.
+        /// For example, 0 (default) will use the front month, 1 will use the back month contract</param>
```

Comment:
> 0 (default)

The default will actually be whatever setting has already been asked by the user really right? Maybe we can just remove `default` to avoid the confusion.